src/main/java/urlshortener/domain/UrlStatus.java

```diff
@@ -4,7 +4,8 @@
   OK("OK"),
   UNSAFE("Unsafe url"),
   INVALID("Invalid url"),
-  UNREACHABLE("Unreachable url");
+  UNREACHABLE("Unreachable url"),
+  IDNOTFOUND("ID of shortened URL not found");
```

Comment:
This isn't a URL Status. It makes no sense to put this here.

Reply:
So I'll create a local variable instead